### PR TITLE
Generate global functions

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/295
Apply https://github.com/gtk-rs/gir/pull/371

### CODE REVIEW

**Comment by EPashkin** [comment]
@GuillaumeGomez Please look at this.
You accept use "namespace.*" as name of object for global functions or better use some other name?

File: src/auto/mod.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (8343e00) from gir-files (71d73f0)
+// This file was generated by gir (5fa90b1+) from gir-files (71d73f0)
```
Comment (GuillaumeGomez) [inline src/auto/mod.rs]: Why the + at the of the hash?

File: Gir.toml [hunk 2]
```diff
@@ -297,6 +297,31 @@ manual = [
     "Pango.WrapMode",
 ]
 
+
```
Comment (GuillaumeGomez) [inline Gir.toml]: Extra empty line.

File: src/auto/mod.rs (hunk 1 above)
Comment (EPashkin) [inline src/auto/mod.rs]: Because it temporary regen in WIP PR ;)

**Comment by GuillaumeGomez** [comment]
Yes, this syntax is fine by me. Don't forget to document it please. :)

File: src/auto/mod.rs (hunk 1 above)
Comment (GuillaumeGomez) [inline src/auto/mod.rs]: This explains that! (thanks for the confirmation)

**Comment by EPashkin** [comment]
Ok.

File: Gir.toml (hunk 2 above)
Comment (EPashkin) [inline Gir.toml]: fixed

**Comment by EPashkin** [comment]
@GuillaumeGomez In this and next related PR's I don't added new reexports and replaced some previously manual function.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**